smoke-test/tests/cypress/integration_test.py

```diff
@@ -165,7 +165,7 @@ def test_run_cypress(frontend_session, wait_for_healthchecks):
     else:
         record_arg = " "
 
-    rest_specs = set(os.listdir("tests/cypress/cypress/integration"))
+    rest_specs = set(os.listdir("tests/cypress/cypress/e2e"))
```

Comment:
why are we changing this

Reply:
Cypress just changed the directory they put these tests in, and they were automatically migrated